main.go

```diff
@@ -33,172 +19,11 @@ func main() {
 
 	log.SetEnableDebugLog(cfg.Verbose)
```

Comment:
will this turn on debug logging in `activatesshkey` package too?
(previously debug logging applied for the functions moved to the `activatesshkey` package)

Reply:
It does as this option is set globally in the log package. This will mean that verbose logging will be enabled also in the project scanner step.